templates/cloud-config-aws-vars.yml

```diff
@@ -0,0 +1,17 @@
+availability_zone1: 
```

Comment:
can we give some random values?

Reply:
ok.. I will provide some random values to indicate how the values should look like.